content/docs/user-guide/interactivity/user-interface/components/interactive/components-draggable.md

```diff
@@ -1,39 +1,44 @@
+![Animation of mouse dragging and dropping UI elements](/images/user-guide/interactivity/user-interface/components/interactive/ui-editor-components-draggable.gif)
 
 To add the draggable component to a UI element, use the **Add Component** menu in the **Properties** pane.
 
 The following picture shows an example of a **Draggable** component, where color has been added to the state actions for **Drag States**.
 
-![UI Editor Draggable component](/images/user-guide/game_ui_editor/ui-editor-components-drag-drop-draggable.png)
+![UI Editor Draggable component](/images/user-guide/interactivity/user-interface/components/interactive/ui-editor-components-drag-drop-draggable.png)
 
-The **Draggable** component is an interactive component. It has the standard [interactive properties](components-interactive-properties).
+The **Draggable** component is an interactive component. It has the standard [interactive properties](properties).
 
 **To edit a Draggable component**
-In the **Properties** pane of the [**UI Editor**](/docs/user-guide/interactivity/user-interface/editor/working), expand **Draggable** and do the following, as appropriate:
+
+In the **Properties** pane of the [**UI Editor**](/docs/user-guide/interactivity/user-interface/editor), expand **Draggable** and do the following, as appropriate:
+
 **Interactable**
-See [Properties](components-interactive-properties) to edit the common interactive component settings.
+
+See [Properties](properties) to edit the common interactive component settings.
+
 **Drag States**
+
 Define the color, alpha, sprite, or font of this element and its child elements for a particular drag state.
 When it is not being dragged, an element with a draggable component uses the interactable states (pause on, pressed, and disabled).
+
 When being dragged, however, the draggable component has an additional three states:
-+ **Normal** - Automatic state when a drag state begins.
 
++ **Normal** - Automatic state when a drag state begins.
    
 + **Valid** - Typically the state used when the draggable component pauses on a valid drop target. This state is determined by a script that you write or C++ code that connects to the `UiDropTargetNotificationBus` and listens for the `OnDropHoverStart` method.
-
    
 + **Invalid** - Typically the state used when a draggable component pauses over an invalid drop target. This state is determined by a script that you write or C++ code. When a valid drag state is triggered, a notification is automatically sent using the `UiDropTargetNotificationBus`.
-
    
 The script or C++ can use the `UiDraggableBus` to set the drag state of the **Draggable** component. It can also set the drop state of the **DropTarget** to indicate valid drop targets to the user.
 To see an example of simple drag Lua script, open `DraggableElement.lua` in `Gems\LyShineExamples\Assets\UI\Scripts\LyShineExamples\DragAndDrop`.
```

Comment:
Remove the indents in these two paragraphs. The indents are causing these sentences to be part of the **Invalid** bullet item paragraph.